### ISSUE TITLE
Carbonite classic couldn't found under twitch client

### ISSUE THREAD

**Issue by sungerbob** [IrcDirk/Carbonite-Classic#42]
Hey, I really like your addon, I hope you keep your interest in.
I can't find classic version in twitch client.
![Twitch](https://user-images.githubusercontent.com/31207969/65463226-1f9b2a00-de60-11e9-9678-480a9a8dc13f.JPG)

**Comment by IrcDirk** [comment]
I think i need to make releases not alpha or beta to make it show on client. Ill check that tommorow.

**Comment by jerj** [comment]
Yeah I think it's good enough to lose the alpha/beta. Any remaining bugs are bound to be minor.

**Comment by IrcDirk** [comment]
Ok. Now its available on twitch/curse client.
